samples/gazell/gzll_ack_payload_host/README.rst

```diff
@@ -1,35 +1,24 @@
-.. _gzll_ack_payload:
+.. _gzll_ack_payload_host:
 
-Gazell ACK Payload
-##################
+Gazell ACK Payload Host
+#######################
 
 .. contents::
    :local:
    :depth: 2
 
-Overview
-********
-
-This sample shows basic Gazell communication and demonstrates how to send payloads and acknowledgments.
-It consists of two applications, one running on the device and one running on the host.
-
-Device
-======
-
-A Device sends a packet and adds a new packet to the TX queue every time it receives an ACK from Host.
-Before adding a packet to the TX queue, the contents of the buttons is copied to the first payload byte (byte 0).
-When the Device receives an ACK, the contents of the first payload byte of the ACK are output to LEDs.
-
-Host
-====
-
-A Host listens for a packet and sends an ACK when it has received the packet.
-The contents of the first payload byte of the received packet is output to LEDs.
-The contents of buttons are sent in the first payload byte (byte 0) of the ACK packet.
+This sample demonstrates the basic Gazell communication and demonstrates how to send packets with payloads and acknowledgments.
+It consists of two Gazell nodes, one running on the Device and one running on the Host.
+Follow the instructions and use of files from this page, and the :ref:`gzll_ack_payload_device` sample.
 
 Requirements
 ************
 
+.. note::
+   Follow the steps from and include the :ref:`gzll_ack_payload_device` sample in addition to this sample.
```

Comment:
```suggestion
   Follow the steps from and include the :ref:`gzll_ack_payload_device` sample to this sample.
```

Reply:
Changes to `" ... sample with this sample"`